src/visualization/plot_recipes.jl

```diff
@@ -119,12 +120,14 @@ PlotData2D(sol::TrixiODESolution; kwargs...) = PlotData2D(sol.u[end], sol.prob.p
 # Note: This is an experimental feature and may be changed in future releases without notice.
 struct PlotDataSeries2D{PD<:PlotData2D}
   plot_data::PD
-  variable_name::String
   variable_id::Int
 end
 
 # Show only a truncated output for convenience (the full data does not make sense)
-Base.show(io::IO, ::PlotDataSeries2D) = print(io, "PlotDataSeries2D(...)")
+function Base.show(io::IO, pds::PlotDataSeries2D)
+  print(io, "PlotDataSeries2D{" * string(typeof(pds.plot_data)) *
+            "}(<plot_data::PlotData2D>, " * string(pds.variable_id) * ")")
```

Comment:
```suggestion
  print(io, "PlotDataSeries2D{", typeof(pds.plot_data),
            "}(<plot_data::PlotData2D>, ", pds.variable_id, ")")
```
It's usually recommended to use the varargs version of `print` instead of creating a string by concatenating various parts at first.